java/AxivityReader.java

```diff
@@ -18,19 +20,25 @@
  */
 public class AxivityReader extends DeviceReader {
 
-    // public static ZonedDateTime SESSION_START = null;
-    // public static ZoneId zoneId = ZoneId.of("Europe/London");
-    public static LocalDateTime SESSION_START = null;
+    private static ZonedDateTime sessionStart = null;
+    private static boolean sessionStartDST;
+
+    private static ZoneId zoneId;
+    private static ZoneRules rules;
+
```

Comment:
Now `SESSION_START` (renamed `sessionStart` for consistency) is a zoned time and its DST status `sessionStartDST` will be registered.` sessionStartDST` is important as all subsequent timestamps' DST status is compared against to see if a crossover has occurred.